Formula/arangodb.rb

```diff
@@ -28,8 +28,11 @@ class Arangodb < Formula
       :revision => "bbe29730e70dba609b57c469e8f863f032fabf3e"
   end
 
+  # fixed upstream in https://github.com/arangodb/arangodb/commit/55ca6c8660e0b373bae1840d61894a4202913cfa
+  patch :DATA
```

Comment:
```suggestion
  patch do
    url "https://github.com/arangodb/arangodb/commit/55ca6c86.diff?full_index=1"
    sha256 "99f653d868360949ee6eb2d5fa380ded3f9a8941caba1ce7b97c88b6e4ae166d"
  end
```

Reply:
This is easier to maintain than having the patch added to the formula itself.

Reply:
Added. Thanks for mention!